exporter/containerimage/annotations.go

```diff
@@ -43,7 +43,7 @@ func ParseAnnotations(data map[string][]byte) (AnnotationsGroup, map[string][]by
 			}
 		}
 
-		switch a.Type {
+		switch exptypes.ImageExporterOptKey(a.Type) {
```

Comment:
This type cast looks wrong

Reply:
Why? It helps us avoid having to `string()` cast all the `case` branches. Even if this isn't a valid key, the switch still matches on the `ImageExporterOptKey` type which will guard from having a `case` with some non-existing option (unless you do `exptypes.ImageExporterOptKey("whatever")`, but that's on you then).

Reply:
... 🤦🏻
Thanks, I looked at the wrong thing 😄